lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/domain/DcsPayload.java

```diff
@@ -21,7 +23,7 @@ public class DcsPayload {
     @JsonProperty private UUID correlationId;
     @JsonProperty private UUID requestId;
     @JsonProperty private String timestamp;
-    @JsonProperty private String passportNumber;
+    @JsonProperty private String drivingLicenceNumber;
```

Comment:
have we decided on drivingLicenceNumber now and not document? Think there may be a few places that it's called documentNumber in the front end